SwiftWisdomTests/StandardLibrary/Numbers/Integer+ExtensionsTests.swift

```diff
@@ -26,7 +26,7 @@ class IntegerExtensionTests: XCTestCase {
     }
     
     func testToMagnitudeString() {      
-        let numbers = [14, 1234, 15420003, 5678000000, 8888000000000, 999999, 7666555444333222, 1234567891234567890]
+        let numbers: [Int64] = [14, 1234, 15420003, 5678000000, 8888000000000, 999999, 7666555444333222, 1234567891234567890]
```

Comment:
I couldn't get unit tests to compile with Xcode 8.2 when I first checked out the repo. The compiler complained that some of the number literals were too big to fit into normal (32-bit?) Ints.